packages/commerce-plugin-tools/src/examples/swell.ts

```diff
@@ -0,0 +1,91 @@
+import { registerCommercePlugin } from '../index';
+registerCommercePlugin(
+  {
+    name: 'Swell',
+    id: '@builder.io/ecome-swell',
```

Comment:
ecome? maybe we can still to our convention of plugin-[name], like `@builder.io/plugin-swell`